src/styles/block/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  @extend %display;
```

Comment:
The name of this extend doesn't clear to me. Maybe %flex-centring would be cleaner
But it's only my opinion)